script/Arcueid/Day 13/13_00_ARC13_1B.txt

```diff
@@ -71,38 +72,42 @@ I pour all my energy into that little finger.
 [sha:aaf9f01f885ef1be7ebc77a339b936bb4a50027c]{
 -- Page 85, Offset 31919.
 -- 小指をわずかに動かす事に全力を尽くす。
-I exert my entire self into moving my pinky ever so slightly.
+I devote every fiber of my being to the goal of simply wiggling my pinky.
 }
 [sha:8ee7df5d5e3ddee638c39f0ef9c5d8a7a2e6abfe]{
 -- Page 86, Offset 31920.
 -- それは破裂しそうな全身の筋肉を、一点に集中させるような作業だった。
-It's as if I'm directing all of my body's close to rupturing muscles into a single point.
+My muscles feel like they're about to burst, but I focus each and every one of them on that single point.
 }
 [sha:92f42e43d0b58bb6128f8e801593f43f3f918e36]{
 -- Page 86, Offset 31921.
 -- 数分が経って、全身が汗だくになって、ようやく小指は微かに反応した。
-Several minutes pass, and with my entire body covered in sweat, my pinky finally responds faintly.
+Several minutes pass. I break out in a sweat. But eventually, my pinky finally twitches slightly.
 }
 [sha:ddaf3601b6a8dad4dbe6536a089f66368a26d050]{
 -- Page 87, Offset 31922.
 -- 「―――、よし」
-"―――Great."
+"―――Alright."
 }
 [sha:7765940f6f339d72cbce0848ea37c24c2a65286e]{
 -- Page 87, Offset 31923.
 -- はあ、と大きく肺から空気を排出する。
-I expel air out of my lungs with a large breath.
+I heave a huge sigh of relief.
 }
 [sha:120a85bb473c9e438c72a8168757915b2c2bdc1c]{
 -- Page 87, Offset 31924.
 -- 小指だけにしろ、体が動くという感覚は頼りになる。自分は人形ではない、動く事ができる、というイメージを取り戻せる。
-// i fucking hat ethis line
-Even if it's only my pinky, I can rely on the feeling of my body moving. I reclaim the mental image of moving my body, affirming myself that I am not a puppet, and that I'm capable of movement.
+// Pretty liberal here.
+// Interpreting 感覚は頼りになる as like breaking through the wall, gaining a foothold
+// Second half rendered more as an internal monologue / psych up.
+Even if it's only my pinky, I've begun to claw back control over my body.
+ I can do this. I'm not just a doll. I can move as I please.
 }
 [sha:34b203eed0d7df7370475ab124dd553a2ab972d0]{
 -- Page 88, Offset 31925.
 -- 次いで、触覚がどんなものかを思い出すように、小指から薬指、手の平、肘、腕、肩、と動く場所を増やしていく。
-Then, as if to remind myself of what it's like to feel, I expand my range of movement from my pinky to my ring finger, palm, elbow, arm, and then my shoulder.
+Next, as though rediscovering my sense of proprioception,
```

Comment:
kinesthesia is a more common term imo

Reply:
Ah, I got that confused with synesthesia